extensions/sql-migration/src/constants/strings.ts

```diff
@@ -410,6 +410,21 @@ export function SQL_TARGET_CONNECTION_SOURCE_NOT_MAPPED(sourceDatabaseName: stri
 		sourceDatabaseName);
 }
 
+//`Database mapping error.  Source database collation ({0}) is not same to target database collation ({1}). Source database: {2}; Target database: {3}.`
+export function SQL_TARGET_SOURCE_COLLATION_NOT_SAME(
+	sourceDatabaseName: string,
+	targetDatabaseName: string,
+	sourceDatabaseCollation: string,
+	targetDatabaseCollation: string): string {
+	return localize(
+		'sql.migration.wizard.target.source.collation.error',
+		"Database mapping error.  Source database collation ({0}) is not same to target database collation ({1}). Source database: {2}; Target database: {3}.",
```

Comment:
I would suggest rewording the string to something like:

"Database mapping error: Source database ({0}) collation ({1}) does not match target database ({2}) collation ({3})."

e.g.

"Database mapping error: Source database (AdventureWorks_source) collation (Latin1_General_CI_AS) does not match target database (AdventureWorks_target) collation (SQL_Latin1_General_CP1_CI_AS)."

Agree with @AkshayMata's comment above to make the message more actionable as well